deepchem/models/torch_models/layers.py

```diff
@@ -1,6 +1,7 @@
 import math
 from math import pi as PI
 import numpy as np
+import sympy as sym
```

Comment:
We need to add this to the appropriate sections of the docs. Try reading through and seeing where we list requirements

Reply:
https://github.com/riya-singh28/deepchem/blob/9f591b0ebcc0a270c480ae397cb821e950c34efc/requirements/env_common.yml

might need to add sympy here as a hard depencency

Reply:
Still need to add it here

Reply:
Had a discussion on this during OH hours and made the suggested changes.

Reply:
The sympy lib is installed before scipy directly in setup.py to ensure the fixed version of scipy after the sympy is installed as sympy installs an older version of scipy.